fix(tIED): Refactor insertIED to avoid duplicate ConnectedAPs

danyill opened the pull request:
Closes #71

Also, renames the `importIED` folder to `insertIED`, hope this is acceptable.

My formatter seems to lint differently but I am not sure why, sorry about this - I did use `npm run format`.

I'd be grateful for a review and happy to make improvements!

JakobVogelsang commented:
Thank you very much. This is a great contribution. I shall have a look.

To resolve the issue with the formatting, you need to `npm install`. Chris has added the prettier as a dependency in the packege.json.

danyill commented:
>To resolve the issue with the formatting, you need to npm install. Chris has added the prettier as a dependency in the packege.json.

Done, thanks for the tip.

> I shall have a look.

As always, gracias.

JakobVogelsang commented on tIED/insertIED.ts:
```diff
@@ -6,6 +6,23 @@ export type InsertIedOptions = {
+ * @param subNetwork - SCL SubNetwork element.
+ * @returns cloned SubNetwork without Element children.
+ */
+function getNewSubNetwork(subNetwork: Element) {
```
The return type definition is missing, isn't it?

danyill replied:
Quite right, fixed

JakobVogelsang reviewed (changes requested):
It seems to me that some of the formatting is not following with what we do in all the other files. @ca-d has added the dependancy `prettier` that does the formating. To use it make sure you have prettier installed on your machine by rebase on main and `npm install` and then run `npm run format`. Other than that the PR is ok.

danyill commented:
I've rebased on main and run `npm run format` which has also modified a few other files by default:

```
        modified:   package-lock.json
        modified:   tExtRef/doesFcdaMeetExtRefRestrictions.spec.ts
        modified:   tExtRef/doesFcdaMeetExtRefRestrictions.ts
        modified:   tExtRef/subscribe.ts
        modified:   tIED/insertIED.ts
```

So I've reverted all the ones except related to this PR.

JakobVogelsang reviewed (approved):
LDTM
